tests/queries/0_stateless/01786_explain_merge_tree.reference

```diff
@@ -0,0 +1,61 @@
+      ReadFromMergeTree
+      ReadType: Default
+      Indexes:
+        None
+          Parts: 5
+          Granules: 12
+        MinMax
+          Parts: 4
+          Granules: 11
+        Partition
+          Parts: 3
+          Granules: 10
+        PrimaryKey
+          Parts: 2
+          Granules: 6
+        Skip
+          Name: t_minmax
+          Parts: 1
+          Granules: 2
+        Skip
```

Comment:
I see `Indexes` header, but this `Skip` looks slightly confusing. Don't have any ideas how to improve naming here.

Reply:
Ok, I will add it

Reply:
Hm, also it may be confused cause I grep out the description.
It will be `minmax GRANULARITY 2`